src/testUtil/TestCaseRecorder.ts

```diff
@@ -248,10 +257,9 @@ export class TestCaseRecorder {
       // command for a navigation map test
       return;
     }
+    await this.testCase!.recordFinalState(returnValue);
```

Comment:
why do we need the `!` now?

Reply:
Got it, fixing. It would be cool to add this to our linters if possible. I guess I had a non-complete `if` check here prior and this was required.

Reply:
Yeah good idea. Looks like [this](https://typescript-eslint.io/rules/no-unnecessary-type-assertion) is the rule? Odd thing is it appears that one should be on by default 🤔